File: app/src/main/java/com/github/bjoernpetersen/q/ui/fragments/LoadingFragment.java
package com.github.bjoernpetersen.q.ui.fragments;


import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.github.bjoernpetersen.q.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class LoadingFragment extends Fragment {


  public LoadingFragment() {
    // Required empty public constructor
  }


  @Override
  public View onCreateView(LayoutInflater inflater, ViewGroup container,
      Bundle savedInstanceState) {
    // Inflate the layout for this fragment
    return inflater.inflate(R.layout.fragment_loading, container, false);
  }

  @Override
  public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
    view.findViewById(R.id.progress).animate();
  }
}

File: app/src/main/java/com/github/bjoernpetersen/q/ui/fragments/LoadingFragment.kt
package com.github.bjoernpetersen.q.ui.fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.github.bjoernpetersen.q.R

/**
 * A simple fragment showing a loading animation.
 */
class LoadingFragment : Fragment() {

  override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
      savedInstanceState: Bundle?): View? =
      inflater?.inflate(R.layout.fragment_loading, container, false)

  override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
    view?.findViewById<View>(R.id.progress)?.animate()
  }
}
